cli/docs/commands/README.md

```diff
@@ -643,13 +643,14 @@ Withdraw FLT collateral from capacity commitment
 ```
 USAGE
   $ fluence delegator collateral-withdraw [IDS] [--no-input] [--env <dar | kras | stage | local | custom>] [--priv-key
-    <private-key>] [--max-cus <value>]
+    <private-key>] [--max-cus <value>] [--finish]
 
 ARGUMENTS
   IDS  Comma separated capacity commitment IDs
 
 FLAGS
   --env=<dar | kras | stage | local | custom>  Fluence Environment to use when running the command
+  --finish                                     Finish capacity commitment after collateral withdrawal
```

Comment:
What is the meaning of it? Why we need to withdraw collateral and not finish commitment?

Reply:
it's for delegators (stackers) so they are not required to do that

Reply:
But stakers most likely will do that via UI and fcli will be used by providers that by default want to finish CC

Reply:
for providers CC is finished by default so it's fine

Reply:
Can you add it in description?

Reply:
where? this command is specifically for delegators so only they will see this flag. Providers don't have this flag and it just does the right thing for them. Delegators have a completely separate set of commands specifically for them (even though most probably they will not use CLI after all)

Reply:
Oh, sorry, I messed up, you're right